rust/crd/src/lib.rs

```diff
@@ -454,6 +453,13 @@ impl TrinoConfig {
             TrinoRole::Coordinator => DEFAULT_COORDINATOR_GRACEFUL_SHUTDOWN_TIMEOUT,
             TrinoRole::Worker => DEFAULT_WORKER_GRACEFUL_SHUTDOWN_TIMEOUT,
         };
+        let requested_secret_lifetime = match role {
+            // TODO: Once Trino supports a HA setup for coordinators we should decrease this!
+            // See https://github.com/stackabletech/trino-operator/issues/693
+            // and https://github.com/stackabletech/decisions/issues/38 for details
+            TrinoRole::Coordinator => Duration::from_days_unchecked(15),
```

Comment:
Nit:
A hint on defaults in the crd description ( e.g. https://github.com/stackabletech/trino-operator/blob/chore/coordinator-restarts/deploy/helm/trino-operator/crds/crds.yaml#L875 ) would be nice.

Reply:
Pls have a look at 030c4ec
And on https://github.com/stackabletech/documentation/pull/701 as well please?

Reply:
Thank you!